renderer-macos/lib/components/ScrollView.re

```diff
@@ -0,0 +1,51 @@
+open Brisk;
+open Layout;
+
+type attr = [ View.attr];
```

Comment:
Style attribute sounded appropriate, because props are for components.

Re square brackets, I thought there are gonna be other `attr`'s, but couldn't come up with anything.

Reply:
so why not `style`?

Reply:
Because `type style = list(attr);`?

Reply:
`Component.style` is a type of `style` like html element style that consists of attributes.

Why are you bothered by this? 😀
By the same logic, `properties` shouldn't be called `props`.

Reply:
`props` are a common name by now. `Attr`s is not. I am bothered by this because again - setting precedence - abbreviated types only cause confusion.

Reply:
It doesn't set precedence, but okay. Want it to be `attribute`?

Reply:
yes 😄